app/model.py

```diff
@@ -104,10 +104,22 @@ class Repo(db.Model):
     existence of a repository.  Key name: unique repository name.  In the UI,
     each repository behaves like an independent instance of the application."""
 
-    # No properties for now; only the key_name is significant.  The repository
-    # title and other settings are all in ConfigEntry entities (see config.py).
-    # The per-repository 'deactivated' setting blocks UI and API access to the
-    # repository, replacing all its pages with a deactivation message.
+    class Status(object):
+        """An enum for the launch/activation status of the repo."""
+        UNLAUNCHED = 0
```

Comment:
"Staging" is the terminology we use in the admin page now. Are you intentionally changing it? I personally prefer "staging" because "unlaunched" can mean that it was launched and then unlaunched.

Reply:
Changed to STAGING.